massbalancemachine/data_processing/transform_to_monthly.py

```diff
@@ -97,6 +97,7 @@ def _get_column_names(vois_topographical: "list[str]") -> "list[str]":
         "POINT_ID",
         "YEAR",
         "N_MONTHS",
+        "PERIOD",  
```

Comment:
Are you sure this is not just a column name specifically for CH data? Otherwise, this might not work for other data (that is supposed to be in the WGMS format).